afctl/parsers.py

```diff
@@ -107,8 +121,15 @@ def config(cls, args):
                 origin = args.o
                 if args.o is None:
                     origin = input("Git origin for deployment : ")
+                if origin != '':
+                    Utility.update_config(config_file, {'global':{'git':{'origin':origin}}})
+
+                token = args.t
+                if args.t is None:
+                    token = input("Git auth token : ")
```

Comment:
why we need this ?

Reply:
To clone a private repository to the cluster. Right now not supported.